core/unit_test/CMakeLists.txt

```diff
@@ -258,6 +258,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;HIP;SYCL)
       SubView_c12
       SubView_c13
       SubView_c14
+      WithoutInitializing
```

Comment:
I think it becomes problematic not to unregister tool hooks after individual unit test when adding to that executable

Reply:
d9ef79802bec7c21cbe408df526bfca17b641ac4